modules/acquia_cms_headless/tests/src/Functional/PureHeadlessModeMenuTest.php

```diff
@@ -101,6 +102,12 @@ public function testChildMenu(string $selector, string $parentMenuName, array $c
     }
   }
 
+  public function testContentModelLinks() {
+    // Make sure alias works fine.
+    $this->drupalGet('/admin/content-models');
+    $this->assertSession()->pageTextContains(' Content Models');
```

Comment:
@chandan-singh7929 remove extra white before text "Content"

Reply:
Done, thank you!